Dockerfile

```diff
@@ -49,7 +49,7 @@ COPY --from=py-ea /usr/bin/elastalert* /usr/bin/
 
 WORKDIR /opt/elastalert-server
 
-COPY package*.json /opt/elastalert-server
+COPY package*.json /opt/elastalert-server/
```

Comment:
Could you discard this?

I've fixed this in https://github.com/Karql/elastalert2-server/commit/32c2e073066bf23a0cb2e58b20dbf37d011c3420